auth-api/src/auth_api/schemas/user.py

```diff
@@ -27,7 +27,7 @@ class Meta:  # pylint: disable=too-few-public-methods
         """Maps all of the User fields to a default schema."""
 
         model = UserModel
-        exclude = ('id',)
+        exclude = ('id', 'orgs')
 
     contacts = fields.Pluck('ContactLinkSchema', 'contact', many=True)
-    orgs = fields.Pluck('MembershipSchema', 'org', many=True)
+    # orgs = fields.Pluck('MembershipSchema', 'org', many=True)
```

Comment:
Can we just remove, or do we need it later?

Reply:
removed